### DISCUSSION TITLE
Sorting based on anything except 'Average'

### DISCUSSION THREAD

**Discussion by dwi2214** [akvorado/akvorado#670] [Q&A]
Hi, 
I would like to ask if it's possible to sort based on dimension like src-address, dst-address etc, and minimum, maximum, or the 95th percentile. 
Currently, it seems that the default sorting is based on 'Average' only, and I am unable to modify that. 

Any explanation would be appreciated. Thank you.

**Comment by vincentbernat** [comment]
Yes, this is not implemented yet.
